src/test/tests/rendering/ospray.py

```diff
@@ -45,7 +45,7 @@ def TestOsprayWindowSave():
     RenderingAtts.osprayShadows = 1
     SetRenderingAttributes(RenderingAtts)
 
-    Test("ospray_window_save")
+    Test("ospray_window_save", pixdiff=0.2, avgdiff=0.01)
```

Comment:
I'm not sure this is needed here since we've never had an issue before. I'll check some more on my end.

Reply:
I understand you've never seen it on Pascal. I think we're just getting lucky there. I definitely see it on Surface and since we know for sure the other test does randomly fail on Pascal, I believe my guess about getting lucky there is correct. I don't think there is any harm in having those thresholds set for this test too.

Reply:
Oh, one other thing, we'll kinda sorta know when the tolerances wind up getting employed to pass an otherwise mistmatching test result...we get a *yellow* color for those cases in the HTML pages. There is also a slightly different diff-status result returned for these cases too, *Acceptable* instead of *None* (indicating no difference).